salt/orchestrate/aws/app_elb.sls

```diff
@@ -40,18 +38,20 @@ create_elb_for_{{ app_name }}_{{ ENVIRONMENT }}:
           enabled: True
           timeout: 300
     - cnames:
+    {% for zone_name, domain in zip(zone_names, domains) %}
```

Comment:
I think we might need to do this as a nested loop since zip will stop as soon as one of the iterables is exhausted, and it won't exhaustively exercise the possible combinations.